pkg/apis/fleet.cattle.io/v1alpha1/bundle.go

```diff
@@ -67,7 +67,8 @@ type BundleSpec struct {
 }
 
 type BundleRef struct {
-	Name string `json:"name,omitempty"`
+	Name           string                `json:"name,omitempty"`
+	BundleSelector *metav1.LabelSelector `json:"bundleSelector,omitempty"`
```

Comment:
I would just use `selector` as the field name, bundle seems redundant.

Reply:
fixed up